R/Tidy_names_types_factors.R

```diff
@@ -267,13 +284,13 @@ parse_date_columns <- function(data, date_formats) {
 
 #' Parse a string to create a named list
 #'
-#' Create a named list from a standardised string of the follwoing format:
+#' Create a named list from a standardized string of the following format:
```

Comment:
Are you suggestion an AE only style convention?

Reply:
reverted